intellij-settings/LiveTemplates.xml

```diff
@@ -108,6 +108,16 @@
   <option name="HTML" value="true" />
 </context>
 </template>
+<template name="cfa:checkbox" value="&lt;th:block th:replace=&quot;~{fragments/inputs/checkbox ::&#10;  checkbox(inputName='$INPUT_NAME$',&#10;  label=#{$LABEL$},&#10; value=#{$VALUE$},&#10; helpText=#{$CHECKBOX_HELP_TEXT$})}&quot;/&gt;" description="A field checkbox with label, name, optional help text and optional icon" toReformat="false" toShortenFQNames="true">
+<variable name="INPUT_NAME" expression="" defaultValue="" alwaysStopAt="true" />
+<variable name="VALUE" expression="" defaultValue="" alwaysStopAt="true" />
+<variable name="LABEL" expression="" defaultValue="" alwaysStopAt="true" />
+<variable name="CHECKBOX_HELP_TEXT" expression="" defaultValue="" alwaysStopAt="true" />
+<variable name="REF_CONTENT" expression="" defaultValue="" alwaysStopAt="true" />
```

Comment:
I suspect this is not needed. Please confirm (Thank you!)

Reply:
It is not, I went ahead and removed it.

Reply:
Thank you so much!